orchestrator/gravity_utils/src/num_conversion.rs

```diff
@@ -1,56 +1,8 @@
-use std::{u128::MAX as U128MAX, u64::MAX as U64MAX};
-
 use clarity::Uint256;
 
-const ONE_ETH: u128 = 1000000000000000000;
-const ONE_ETH_FLOAT: f64 = ONE_ETH as f64;
-pub fn one_eth() -> Uint256 {
-    ONE_ETH.into()
-}
-
-const ONE_GWEI: u128 = 1000000000;
-const ONE_GWEI_FLOAT: f64 = ONE_GWEI as f64;
-pub fn one_gwei() -> Uint256 {
-    ONE_GWEI.into()
-}
-
-const ONE_NOM: u128 = 1000000000000000000;
-const ONE_NOM_FLOAT: f64 = ONE_NOM as f64;
-pub fn one_nom() -> Uint256 {
-    ONE_NOM.into()
-}
-
-pub fn downcast_uint256(input: Uint256) -> Option<u64> {
-    if input >= U64MAX.into() {
-        None
-    } else {
-        let mut val = input.to_bytes_be();
-        // pad to 8 bytes
-        while val.len() < 8 {
-            val.insert(0, 0);
-        }
-        let mut lower_bytes: [u8; 8] = [0; 8];
-        // get the 'lowest' 8 bytes from a 256 bit integer
-        lower_bytes.copy_from_slice(&val[0..val.len()]);
-        Some(u64::from_be_bytes(lower_bytes))
-    }
-}
-
-pub fn downcast_to_u128(input: Uint256) -> Option<u128> {
-    if input >= U128MAX.into() {
-        None
-    } else {
-        let mut val = input.to_bytes_be();
-        // pad to 8 bytes
-        while val.len() < 16 {
-            val.insert(0, 0);
-        }
-        let mut lower_bytes: [u8; 16] = [0; 16];
-        // get the 'lowest' 16 bytes from a 256 bit integer
-        lower_bytes.copy_from_slice(&val[0..val.len()]);
-        Some(u128::from_be_bytes(lower_bytes))
-    }
-}
+const ONE_ETH_FLOAT: f64 = 1000000000000000000.;
+const ONE_GWEI_FLOAT: f64 = 1000000000.;
+const ONE_NOM_FLOAT: f64 = 1000000.;
```

Comment:
Why the value for ONE_NOM is not "1000000000000000000" as it was before?

Reply:
I must have merged the wrong way, I will fix it